org.kde.kalgebra.json

```diff
@@ -0,0 +1,37 @@
+{
+    "id": "org.kde.kalgebra",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "base": "io.qt.qtwebengine.BaseApp",
+    "base-version": "5.15-21.08",
+    "command": "kalgebra",
+    "rename-icon": "kalgebra",
+    "copy-icon": true,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri"
+    ],
+    "separate-locales": false,
+
+    "modules": [
+	"analitza.json",
```

Comment:
same comment here: just put it all in the same .json.

Reply:
Sure, updated in latest commit.